addons/l10n_mx/data/account_configuration.yml

```diff
@@ -0,0 +1,10 @@
+-
+  Account settings set for Use Anglo-Saxon Accounting
+-
+  !record {model: account.config.settings, id: mx_default_account_properties}:
+    {
+      'use_anglo_saxon': 1,
```

Comment:
For Pa and MX it works and for Spain also (if you set no real time valuation which is the way to work in spain) I do not see any problem, I have it in my production environment on this way.